# Check if gripper is calibrated

---
bool calibrated
bool success
string msg
